### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by PHaeJin**
Fix some broken links

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -72,7 +72,7 @@
 explanatory text. The Notebook has support for multiple programming
 languages, sharing, and interactive widgets.
 
-Read `the documentation <https://jupyter-notebook.readthedocs.io>`_
+Read `the documentation <https://jupyter-notebook.readthedocs.io/en/stable/>`_
```
Comment (gnestor) [inline setup.py]: What if we support other languages in the future? I think readthedocs can potentially redirect to the appropriate labguage version based on locale. I think we would be better off sticking with the original.

File: docs/source/links.txt [hunk 2]
```diff
@@ -25,8 +25,8 @@
 .. _notebook_p1: https://nbviewer.jupyter.org/github/jupyter/notebook/blob/master/docs/source/examples/Notebook/Running%20Code.ipynb
 
 .. Other python projects
-.. _matplotlib: http://matplotlib.org
-.. _nbviewer: http://nbviewer.jupyter.org
+.. _matplotlib: http://matplotlib.org/
+.. _nbviewer: http://nbviewer.jupyter.org/
```
Comment (gnestor) [inline docs/source/links.txt]: 🤷‍♂️

File: setup.py (hunk 1 above)
Comment (PHaeJin) [reply setup.py]: Alright. Fixing it now!

**Comment by gnestor** [comment]
Thanks @PHaeJin!

**Pull request merged**